python/MaterialXTest/tests_to_html.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+
+import sys
+import getopt
+import os
+import datetime
+
+try:
+    # Use pip to install Pillow and Image to enable image diffs
+    from PIL import Image
+    from PIL import ImageChops
+    DIFF_ENABLED = True
+except:
+    DIFF_ENABLED = False
+
+def createDiff(image1Path, image2Path, imageDiffPath):
```

Comment:
As mentioned this should be a user defined command. It's also not hard to write our own command.

Reply:
I agree with this, was going to do that as a separate code review, is that ok or did you want it as part of this one?

Reply:
No. That's fine. It's just a comment.